doc/managing_pachyderm/upgrades.md

```diff
@@ -1,116 +1,102 @@
-# Upgrades
+# Upgrade Pachyderm to a Minor Version
```

Comment:
```suggestion
# Upgrade Pachyderm Minor Versions
```
I don't think you're really upgrading "to" a minor version but "between minor versions"?

Also, I wonder if we shouldn't use the term "minor" or "major" version at all since we're not actually following the correct definitions of those in semantic versioning. I dont actually feel strongly about any of this, just a minor thought

Reply:
I changed to "Upgrade Pachyderm".